README.md

```diff
@@ -0,0 +1,23 @@
+# Pricing app
+
+## I. Setup and run
+
+### 1. Backend
+
+```bash
+cd pricing-backend
+npm install
+npm start
```

Comment:
Good job having an instruction. In nextjs you can also have server rendering and don't require a separated backend right?

Reply:
Yes. I have already setup a separate backend and implement it, after that, I know nextjs also have server rendering. I will try to set up that way. Thank you for your suggestion!

Reply:
Yes. I have already setup a separate backend and implement it, after that, I know nextjs also have server rendering. I will try to set up that way. Thank you for your suggestion!